src/console/Microsoft.Health.Fhir.Ingest.Console.Common/Extensions/FhirTransformationExtensions.cs

```diff
@@ -40,6 +39,8 @@ public static IServiceCollection AddNormalizedEventReader(this IServiceCollectio
                 // Add EventHubClientOptions (Used for EventProcessorClient - common to all applications)
                 var options = new EventHubClientOptions();
                 config.GetSection("NormalizationEventHub").Bind(options);
+                var tokenProvider = sp.GetService<IAzureCredentialProvider>();
+                options.EventHubTokenCredential = tokenProvider?.GetCredential();
```

Comment:
Same as other comment, not sure about including tokens inside the options class.